lib/person_cards.rb

```diff
@@ -23,6 +23,14 @@ def image
     person.image
   end
 
+  def legislature
+    term.legislature
+  end
+
+  def top_identifiers
+    term.top_identifiers
+  end
+
```

Comment:
Ah, no - it's used by the term_table.erb view.

@tmtmtmtm I think / hope this is OK now!